packages/suite/src/components/wallet/CoinmarketPaymentType/index.tsx

```diff
@@ -30,14 +31,22 @@ interface Props {
 
 const CoinmarketPaymentType = ({ method }: Props) => (
     <Wrapper>
-        {!method && <Text>Unknown payment method</Text>}
+        {!method && (
+            <Text>
+                <Translation id="TR_PAYMENT_METHOD_UNKOWN" />
+            </Text>
+        )}
         {method && (
-            <IconWrapper>
-                <Icon
-                    width="40px"
-                    src={`${invityApi.server}/images/paymentMethods/${method}.svg`}
-                />
-            </IconWrapper>
+            <>
+                <IconWrapper>
+                    <Icon
+                        width="24px"
+                        src={`${invityApi.server}/images/paymentMethods/suite/${method}.svg`}
+                    />
+                </IconWrapper>
+                {/* temporary solution - payment mehtod name will be returned by API server to be independent on translations */}
+                <Translation id={`TR_PAYMENT_METHOD_${method.toUpperCase()}` as any} />
```

Comment:
maybe you should create a switch returning proper translation id since you will need it in anyway. 
example:

https://github.com/trezor/trezor-suite/blob/29c58f95436f114a51d44239708eeb00a685b3ee/packages/suite/src/utils/suite/device.ts#L63-L87

Reply:
@szymonlesisz Actually, in very near future we will not need the translations at all as we plan to update our API server to return translated payment method names. We want to be able to add products without the necessity to do changes in the Suite code.

For this reason I used this not totally clean method. The server will return only correct payment methods and even if not, the damage is minimal (the use will see unknown key xyz).

It seems to me a waste of time to create huge switch statement.

Reply:
but this way names like: "Bank Transfer" or "Credit card" will never be translated, no?

Reply:
We will pass user's language choice to the server, the server will return translated texts.

Reply:
ah i get it now